docs/user-guide/examples/advanced_examples/advanced_orchestration.md

```diff
@@ -1,10 +1,362 @@
+
+Our final distribution for modeling the anticipated need for each level of charging port in our _k = 100_ EV garage can be written as:
+
+_L ~ P(k,λ)_
+
+_λ ~ N(μ,σ)_
+
+<center>
+
+|     | _L1_     |   _L2_   |           _L3_ |
+| --- | :------- | :------: | -------------: |
+| _μ_ | 0.3      |   0.5    |            0.2 |
+| _σ_ | ~ N(1,3) | ~ N(1,2) | ~ N(0.05,0.25) |
+
+</center>
+
+![](../../../img/EVfulldist.png)
+
+Notice that the individual overplotted distributions in the two histograms above are different -- there is more flexibility encoded into the second. The distributions in the second plot describe the following assumptions about the anticipated need for Level 1, 2, and 3 chargers:
+
+1. The number of charging ports is discrete and positive (Poisson).
+2. The variance in the number of ports should be normally distributed.
+3. The average percentage of Level 1, 2, and 3 chargers is 30%, 50%, and 20%, respectively.
+4. The variance around the average is uniformly distributed.
+5. The variance is relatively very broad for Level 1, broad for Level 2, and very constrained for Level 3. This means we are very confident in our guess of the average percentage for Level 3 chargers, less confident for Level 2, and much less confident for Level 3.
+
+We have described the individual distributions for each level of charging port. What we don't know, prior to running the Monte Carlo co-simulation, is how these distributions will jointly impact the research question.
+
+#### Research Question Configuration
+
+We want to address the research question: What is the likely power draw that EVs will demand?
+
+At the beginning of the co-simulation, the distributions defined above will be sampled _N_ times within the EV federate, where _N =_ the number of parking spots/charging ports in the garage. The output of the initial sampling is the number of requested Level 1, 2, and 3 charging ports. The soc for the batteries on board are initialized to a uniform random number between 0.05 and 0.5, and these SOC are sent to the Charger federate. If the SOC of an EV battery is less than 0.9, the EV Controller federate tells the EV battery in the EV federate to continue charging. Otherwise, the EV Charger disconnects the EV battery, and instructs the it to sample a new EV battery from the distributions (one sample).
+
+After the two federates pass information between each other -- EV Battery sends SOC, EV Charger instructs whether to keep charging or resample the distributions -- the EV Battery calculates the total power demanded in the last time interval.
+
+## Execution and Results
+
+Execution can be done with either a simple script (provided on the repo), or with Merlin.
+
+### Manual Orchestration Execution
+
+Manual implementation of the co-simulation is done with the helper script `advanced_orchestration.py`, with command line execution:
+
+```
+$ python advanced_orchestration.py
```

Comment:
@allisonmcampbell the `advanced_orchestration.py` file doesn't exist in https://github.com/GMLC-TDC/HELICS-Examples/tree/main/user_guide_examples/advanced/advanced_orchestration

Reply:
I think I figured this one out -- the `master` branch in HELICS-Examples has been renamed to `main`; make sure you update your forks to push changes to `main` instead of `master`. I'm going to merge the changes you made into `main` and do some branch renaming to try to fix the automatic redirects GitHub sets up.

Reply:
Change is merged -- redirect is working again for links.

Remember to update your fork to use `main` instead of `master`; the instructions GitHub gives for doing this when you visit https://github.com/GMLC-TDC/HELICS-Examples for the first time after it has been renamed are:

```
git branch -m master main
git fetch origin
git branch -u origin/main main
git remote set-head origin -a
```

GitHub should also show this message if you push to the wrong branch: `remote: Heads up! The branch 'master' that you pushed to was renamed to 'main'.`

Reply:
Yes, thank you for the reminder!